Content.Shared/Pinpointer/PinpointerComponent.cs

```diff
@@ -12,8 +12,24 @@ namespace Content.Shared.Pinpointer;
 public sealed partial class PinpointerComponent : Component
 {
     // TODO: Type serializer oh god
-    [DataField("component"), ViewVariables(VVAccess.ReadWrite)]
-    public string? Component;
+    /// <summary>
+    ///     A list of components that will be searched for when selected from the verb menu.
+    ///     The closest entities found with one of the components in this list will be added to the StoredTargets.
+    /// </summary>
+    [DataField("components"), ViewVariables(VVAccess.ReadWrite)]
+    public List<string> Components = new();
+
```

Comment:
As per discord datafields do not require strings anymore.

Also ComponentRegistry not a string list.